src/rebar_prv_common_test.erl

```diff
@@ -70,6 +71,19 @@ format_error({failures_running_tests, FailedCount}) ->
 format_error({error_running_tests, Reason}) ->
     io_lib:format("Error running tests: ~p", [Reason]).
 
+run_test(CTOpts, true) ->
+    handle_results(ct:run_test(CTOpts));
+run_test(CTOpts, false) ->
+    Pid = self(),
+    LogDir = proplists:get_value(logdir, CTOpts),
+    erlang:spawn(fun() ->
+        {ok, F} = file:open(filename:join([LogDir, "ct.latest.log"]),
+                             [write]),
+        true = group_leader(F, self()),
+        Pid ! ct:run_test(CTOpts)
+    end),
+    receive Result -> handle_quiet_results(CTOpts, Result) end.
```

Comment:
So uh what happens if we crash? Hanging forever? This needs a monitor or a link or something.